Gemfile

```diff
@@ -27,7 +27,7 @@ group(:features) do
   gem 'hocon', '~> 1.0', require: false
   # requires native libshadow headers/libs
   #gem 'ruby-shadow', '~> 2.5', require: false, platforms: [:ruby]
-  gem 'minitar', '~> 0.9', require: false
+  gem 'minitar', '~> 1.0', require: false
```

Comment:
Is it required to list it here if it's also in the gemspec? Is that something special because it's a feature?

Reply:
We only add minitar as a hard runtime dependency when building windows specific puppet gems, for example, compare the runtime dependencies in https://rubygems.org/gems/puppet/versions/8.8.1 and https://rubygems.org/gems/puppet/versions/8.8.1-x64-mingw32. So it's a hard dependency on windows but soft on other platforms. And soft dependencies are managed via puppet's feature system, e.g. `Puppet.features.minitar?`